application/features/password/password.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Password
+  In order to update the password of a specific user
+  As an authorized user
+  I need to be able to update a specific user password
+
+  Scenario: Provide a new password for an existing user
+    Given I receive an existing employee id
+      And the requestor is authorized
+      And the user does not already have a password
+      And I receive a password
+    When I receive a request to create a password for a specific user
+    Then a new password hash should be stored
+      And the last changed date should be stored as the instant it was stored
+      And the last changed date should be stored in UTC
+      And the last synched date should be stored as the instant it was stored
```

Comment:
ok, sounds like something I'd like to talk through a bit more...especially considering some of the redesign @forevermatt and I have been contemplating around `POST` and `PUT`.  Adding a reminder:  https://github.com/silinternational/idp-id-broker/commit/53ee077cde3d9bbc0f1de4898545198d4ecd756b